src/routes/reference/jsx-attributes/innerhtml.mdx

```diff
@@ -1,5 +1,18 @@
 ---
 title: innerHTML
+use_cases: >-
+  rendering html strings, dynamic html content, third-party html, legacy content
+  migration, sanitized markup
+tags:
+  - html
+  - dom
+  - security
+  - content
+  - markup
+version: '1.0'
+description: >-
+  Set raw HTML content in SolidJS elements using innerHTML attribute. Render
+  HTML strings dynamically with security considerations for user data.
```

Comment:
Can you please remove "with security considerations for user data". I tried to edit it myself but doesnt allow me